<launch>

<!-- Arguments required by the node -->
<!-- No of regions of interests to be extracted -->
<!-- Maximum number of object to be found in each ROI -->
<!-- HSV-Limits Configuration Files with full path -->
<!-- Region Labes used by the nodes to publish the transforms -->

	<node name="poseEstimator6D" pkg="tower_of_hanoi_sdk" type="poseEstimator6D" output="screen" args="3 3 $(find tower_of_hanoi_sdk)/redRoiConfig.cfg $(find tower_of_hanoi_sdk)/greenRoiConfig.cfg $(find tower_of_hanoi_sdk)/yellowRoiConfig.cfg red green yellow">
	<rosparam command="load" file="$(find tower_of_hanoi_sdk)/cfg/poseEstimatorConfiguration.yaml" />
	</node>
</launch>
